src/components/Navigation/Link/Link.test.js

```diff
@@ -1,31 +1,31 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import { NavLink } from 'react-router-dom';
+import { fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
```

Comment:
Same here

Reply:
In order to use the matcher `toHaveAttribute` you need to import `@testing-library/jest-dom`. You can check more about it [here](https://github.com/testing-library/jest-dom).